bazel/external/wee8.genrule_cmd

```diff
@@ -51,8 +51,17 @@ if [[ $${ENVOY_TSAN-} == "1" ]]; then
   WEE8_BUILD_ARGS+=" is_tsan=true"
 fi
 
-# Release build.
-WEE8_BUILD_ARGS+=" is_debug=false"
+# Debug/release build.
+if [[ $(COMPILATION_MODE) == "dbg" && $${ENVOY_UBSAN_VPTR-} != "1" && $${ENVOY_MSAN-} != "1" && $${ENVOY_TSAN-} != "1" ]]; then
+  WEE8_BUILD_ARGS+=" is_debug=true"
+  WEE8_BUILD_ARGS+=" v8_symbol_level=2"
+  WEE8_BUILD_ARGS+=" v8_optimized_debug=false"
```

Comment:
compile_time_options is a dbg build. This is the cause, the test doesn't fail on opt or fastbuild, but dbg.

Reply:
The test fails because the wee8 version is different.  The check is IsSupportedVersion in wee8/src/wasm/wasm-serialization.cc 

Attached is a patch which demostrates it.